service/chartconfig/v1/helm/helm_test.go

```diff
@@ -200,3 +200,63 @@ func Test_GetReleaseHistory(t *testing.T) {
 		})
 	}
 }
+
+func Test_InstallRelease(t *testing.T) {
+	testCases := []struct {
+		description   string
+		namespace     string
+		releases      []*helmrelease.Release
+		expectedError bool
+	}{
+		{
+			description: "basic install, empty releases",
+			namespace:   "default",
+			releases:    []*helmrelease.Release{},
+		},
+		{
+			description: "other release in same namespace",
+			namespace:   "default",
+			releases: []*helmrelease.Release{
+				helmclient.ReleaseMock(&helmclient.MockReleaseOptions{
+					Name:      "my-chart",
+					Namespace: "default",
+				}),
+			},
+		},
+		{
+			description: "same release in same namespace",
+			namespace:   "default",
+			releases: []*helmrelease.Release{
+				helmclient.ReleaseMock(&helmclient.MockReleaseOptions{
+					Name:      "test-chart",
+					Namespace: "default",
+				}),
+			},
+			expectedError: true,
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.description, func(t *testing.T) {
+			helm := Client{
+				helmClient: &helmclient.FakeClient{
+					Rels: tc.releases,
+				},
+				logger: microloggertest.New(),
+			}
+			// helm fake client does not actually use the tarball
```

Comment:
Nitpick but this should be a sentence with a `.`.

Reply:
Sure, done.